src/main/java/com/RaiseTech8/MyBatisdemo/Name.java

```diff
@@ -0,0 +1,25 @@
+package com.RaiseTech8.MyBatisdemo;
```

Comment:
Javaではパッケージ名はアルファベットは小文字のみを使います。
```suggestion
package com.raisetech8.mybatisdemo;
```

Reply:
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html
> The prefix of a unique package name is always written in all-lowercase ASCII letters

Reply:
43e9f557c56f9875aff2c32c04c1d2712fa31aca
- パッケージ名の変更を行いました。

必要のない差分を出してしまいました、、
IntelliJ画面でのdiff表示の色の意味をやっと理解したのでpush前に必ず確認するようにします。